src/app/borrow/pages/borrow-landing/borrow-landing.page.ts

```diff
@@ -51,47 +53,28 @@ export class BorrowLandingPage implements OnInit {
     private wallet: WalletProviderService,
     private priceFeed: PriceOracleService,
     private vaultService: CheddaBaseTokenVaultService,
+    private vaultStatsService: VaultStatsService
+    
     ) { }
   
   async ngOnInit() {
-    this.lendingPools = environment.config.pools
+    this.currency = environment.config.networkParams.nativeCurrency.name
     this.vaultContract = this.vaultService.contractAt(environment.config.contracts.CheddaBaseTokenVault)
-    await this.loadVaultStats()
+    await this.vaultStatsService.loadVaultStats()
```

Comment:
You can remove all these calls to `loadVaultStats()` and just call them in the event listeners. Currently the stats are not reloaded on borrow and repayment of loan.

Reply:
The stats are reloaded on the borrow landing page on my side. I just verified that. 
It needs a little time to update the stats on the landing. We might have to put a loading indicator.